File: Problem 10/ans.c
#include<stdio.h>

int prime(long long int x)
{
    int a;
    for(a=2;a<x;a++)
        if (!(x%a))
            return 0;
    return 1;
}

int main()
{
    long long int sum=0,a;
    for (a=2;a<2000000;a++)
        if (prime(a))
               sum+=a;
    printf("Sum of primes <20000000: %lld",sum);
    return 0;
}

File: Problem 50/ans.py
import sympy

leng=0
num=0
x=list(sympy.sieve.primerange(2,1000000))
for i in range(len(x)-1):
    for j in range(i+1,len(x)):
        if (sum(x[i:j+1])>1000000):
            break
        if (sum(x[i:j+1]) in x) and (j-i)>=leng and sum(x[i:j+1])>num:
            leng=j-i
            num=sum(x[i:j+1])
print("Largest prime under 1 million:",num)